src/celesto/computer.py

```
@@ -257,7 +258,33 @@ def recv_loop():
+                        elif comp_status == "running":
+                            console.print(f"[yellow]Computer is still running. Reconnect with: celesto computer ssh {computer_id}[/yellow]")
+                        elif comp_status in ("stopped", "stopping"):
+                            console.print(f"[yellow]Computer is {comp_status}. Start it with: celesto computer start {computer_id}[/yellow]")
+                        elif comp_status in ("deleted", "deleting"):
+                            console.print("[red]Computer has been deleted.[/red]")
+                            console.print("[dim]Create a new computer with: celesto computer create[/dim]")
+                        elif comp_status in ("creating", "starting"):
+                            console.print(f"[yellow]Computer is {comp_status}. Wait a moment and retry: celesto computer ssh {computer_id}[/yellow]")
+                        else:
+                            console.print(f"[yellow]Computer status: {comp_status}[/yellow]")
+                except (CelestoNotFoundError,):
```

Ruff (0.15.9)

[warning] A length-one tuple literal is redundant in exception handlers

Replace with `except CelestoNotFoundError`

(B013)